runtime.md

```diff
@@ -72,7 +72,7 @@ This operation MUST return the state of a container as specified in the [State](
 This operation MUST generate an error if it is not provided a path to the bundle and the container ID to associate with the container.
 If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST generate an error and a new container MUST not be created.
 Using the data in [`config.json`](config.md), this operation MUST create a new container.
-This means that all of the resources associated with the container MUST be created, however, the user-specified process MUST NOT be run at this time.
+This means that all of the resources associated with the container MUST be created, however, the user-specified code (from [`process`][process]) MUST NOT be run at this time.
```

Comment:
On Fri, May 27, 2016 at 06:50:14PM -0700, Qiang Huang wrote:
> > -This means that all of the resources associated with the container MUST be created, however, the user-specified process MUST NOT be run at this time.
> > +This means that all of the resources associated with the container MUST be created, however, the user-specified code (from [`process`][process]) MUST NOT be run at this time.
>
> `user-specified code` seems a bit abstract to me, and I don't see
> the problem with the original one.

The distinction between “process” and “code” is that fork(2) and
similar create a new process running the caller's code, while
execve(2) and similar execute new code in the caller's existing
process.  In this spec, the container *process* is created at
create-time (with fork(2) or similar) and then that process is
switched to user-specified code at start-time (with execve(2) or
similar).  I'm dropping the “process MUST NOT be run” phrasing here,
because we *do* want to create the container *process* here.  We just
don't want it running the user's code yet.